service/payments/src/test/java/nl/tudelft/wdm/group1/payments/events/ConsumerTest.java

```diff
@@ -1,13 +1,13 @@
 package nl.tudelft.wdm.group1.payments.events;
 
-import nl.tudelft.wdm.group1.common.*;
+import nl.tudelft.wdm.group1.common.model.Order;
+import nl.tudelft.wdm.group1.common.model.Payment;
```

Comment:
Wouldn't it be better to call the package `models` instead of `model`? Makes it more consistent with `topics`

Reply:
Same for `exception`. Or, to make it easier, you could also rename `topics` to `topic`